modules/how-to-plan-your-environment-according-to-application-requirements.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/planning-your-environment-according-to-object-limits.adoc
+
+[id='how-to-plan-according-to-application-requirements']
```

Comment:
All the anchors in modules need `{context}` in them, like [id='how-to-plan-according-to-application-requirements-{context}']